client/components/mma/delivery/records/ProductDetailsTable.tsx

```diff
@@ -90,11 +84,11 @@ export const ProductDetailsTable = (props: ProductDetailsTableProps) => {
 			<dl css={dlCss}>
 				<div>
 					<dt>Product:</dt>
-					<dd>{props.productName}</dd>
+					<dd data-qm-masking="blocklist">{props.productName}</dd>
```

Comment:
same as above, wouldn't this just be like "Newspaper Delivery"

Reply:
good point